[2023-05-05 01:57:31,698] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:2210 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:57:31,701] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: file_sensor_example.copy_file scheduled__2023-05-05T00:00:00+00:00 [queued]>
[2023-05-05 01:57:31,710] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: file_sensor_example.copy_file scheduled__2023-05-05T00:00:00+00:00 [queued]>
[2023-05-05 01:57:31,711] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 01:57:31,711] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-05-05 01:57:31,711] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 01:57:31,722] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): copy_file> on 2023-05-05 00:00:00+00:00
[2023-05-05 01:57:31,725] {standard_task_runner.py:52} INFO - Started process 5728 to run task
[2023-05-05 01:57:31,728] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'file_sensor_example', 'copy_file', 'scheduled__2023-05-05T00:00:00+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/sensor_sample.py', '--cfg-path', '/tmp/tmp1v8kk4mb', '--error-file', '/tmp/tmpjx68i70l']
[2023-05-05 01:57:31,729] {standard_task_runner.py:77} INFO - Job 143: Subtask copy_file
[2023-05-05 01:57:31,798] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dagrun.py:466 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:57:31,804] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dagrun.py:466 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:57:31,813] {logging_mixin.py:109} INFO - Running <TaskInstance: file_sensor_example.copy_file scheduled__2023-05-05T00:00:00+00:00 [running]> on host ebd7ac88a2d5
[2023-05-05 01:57:31,839] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dagrun.py:466 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:57:31,845] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dagrun.py:466 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:57:31,853] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/xcom.py:323 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:57:31,873] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:126 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:57:31,883] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:162 SAWarning: Coercing Subquery object into a select() for use in IN(); please pass a select() construct explicitly
[2023-05-05 01:57:31,896] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:163 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:57:31,901] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=file_sensor_example
AIRFLOW_CTX_TASK_ID=copy_file
AIRFLOW_CTX_EXECUTION_DATE=2023-05-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-05T00:00:00+00:00
[2023-05-05 01:57:31,902] {logging_mixin.py:109} INFO - test_file.txt already exists in /app/datalake/bronze/sensor_sample
[2023-05-05 01:57:31,903] {logging_mixin.py:109} INFO - test_file2.txt already exists in /app/datalake/bronze/sensor_sample
[2023-05-05 01:57:31,904] {logging_mixin.py:109} INFO - test_file3.txt already exists in /app/datalake/bronze/sensor_sample
[2023-05-05 01:57:31,904] {logging_mixin.py:109} INFO - test_file4.txt already exists in /app/datalake/bronze/sensor_sample
[2023-05-05 01:57:31,912] {logging_mixin.py:109} INFO - Copied test_file5.txt to /app/datalake/bronze/sensor_sample
[2023-05-05 01:57:31,913] {python.py:175} INFO - Done. Returned value was: None
[2023-05-05 01:57:31,947] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=file_sensor_example, task_id=copy_file, execution_date=20230505T000000, start_date=20230505T015731, end_date=20230505T015731
[2023-05-05 01:57:31,980] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-05-05 01:57:32,035] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-05 01:57:32,037] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dag.py:1076 SADeprecationWarning: Query.value() is deprecated and will be removed in a future release.  Please use Query.with_entities() in combination with Query.scalar() (deprecated since: 1.4)
[2023-05-05 01:57:32,038] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dag.py:1076 SADeprecationWarning: Query.values() is deprecated and will be removed in a future release.  Please use Query.with_entities() (deprecated since: 1.4)
